### PR TITLE
Add part 1 of Embedded Ansible posts authored by Peter McGowan

### PR DESCRIPTION
**Pull request by d-m-u**
Peter wrote 4 articles about embedded Ansible work in Fine that were intended for posting on the CloudFormsNow site but aren't going to all get posted and so I was asked to put them here since it's a shame to waste the research and time that went into this effort. They're great, and we should post them...

### CODE REVIEW

**Comment by d-m-u** [comment]
hey @cybette could I get your 👀 please?

**Comment by d-m-u** [comment]
@pemcg please review

File: site/_posts/2020-02-12-Embedded-Ansible-Part-1-Built-In-Roles.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+title: "Embedded Ansible — Part 1, Built-In Ansible Roles"
+author: Peter McGowan
+date: 2020-02-12
+comments: true
+published: true
+---
```
Comment (cybette) [inline site/_posts/2020-02-12-Embedded-Ansible-Part-1-Built-In-Roles.md]: Perhaps add this to the front matter?
`tags: tutorials`

**Comment by cybette** [comment]
This is great content, thanks @d-m-u and @pemcg!

Please add a note about Cloudforms vs ManageIQ, and for this being available in Cloudforms 4.5 = ManageIQ Fine release.

**Comment by pemcg** [comment]
This looks great. The only comment that I'd make is maybe just make it purely about ManageIQ and remove any reference to 'CloudForms' (as it's intended for the ManageIQ blog)? Is there a way of retaining the screenshot images rather than replace with a table? I can probably re-create the images as PNGs ion it helps?

**Comment by cybette** [comment]
Thanks @pemcg! Yes it's better to make it purely ManageIQ if everything applies upstream without modification. PNGs will work best although JPGs are fine too (I'm not sure what the original format is.)

**Comment by cybette** [comment]
@d-m-u  LGTM! :+1:

**Comment by pemcg** [comment]
LGTM 👍

**Pull request merged**